dev-environment/docker-compose.yml

```diff
@@ -4,19 +4,56 @@ services:
 
   mlflow:
     build: ./mlflow
-    container_name: mlflow_server
+    container_name: mlflow-server
     restart: always
     ports:
-     - "5000:5000"
+      - "5000:5000"
```

Comment:
Let's add YAML checker at some point to prevent formatting changes.

Reply:
ok, good point, will remember